src/main/resources/com/puppycrawl/tools/checkstyle/meta/checks/coding/DeclarationOrderCheck.xml

```diff
@@ -45,10 +45,10 @@
  &lt;/pre&gt;</description>
          <properties>
             <property default-value="false" name="ignoreConstructors" type="boolean">
-               <description>control whether to ignore constructors.</description>
+               <description>Control whether to ignore constructors.</description>
             </property>
             <property default-value="false" name="ignoreModifiers" type="boolean">
-               <description>control whether to ignore modifiers (fields, ...).</description>
+               <description>Control whether to ignore modifiers (fields, ...).</description>
```

Comment:
If we are making improvements,  we need to reword this description, it is nonsense. `Configures check to ignore modifiers on target tokens when considering declaration order.` or something is much better.

Reply:
Technically we are not improving. 
We just migrate to macros as it, and this is just minor side effect.

I can do one more commit to change wording. As you suggested.

Reply:
Ok, I did not realize change in capitalization was from generation, we don’t need to do this.

Reply:
please proceed with review.